### ISSUE TITLE
Usage of namespaces causes "unexpected token" error when PHPMD is installed via Phive

### ISSUE THREAD

**Issue by joergmoldenhauer** [phpmd/phpmd#960]
- PHPMD version: 2.9.1
- PHP Version: 8.1.1
- Installation type: Phive
- Operating System / Distribution & Version: Alpine 3.15

### Current Behavior
When I declare a namespace in a class or use a class from another namespace I get an "unexpected token" error. This only happens, when PHPMD is installed via Phive. If it is installed via Composer there is no error.

### Expected Behavior
Analysis should work without error when PHPMD is installed with Phive.

### Steps To Reproduce:
* Install PHPMD with Phive: `phive install phpmd`
* Create a class in a namespace or add a use statement for a class in another namespace
* Execute `tools/phpmd src text phpmd.xml`
* You should see something like: `Unexpected token: App\Foo, line: 5, col: 11, file: /var/www/html/src/Foo/Foo.php`

My example class looks like this:
```php
<?php

declare(strict_types=1);

namespace App\Foo;

use App\Bar;

final class Foo
{

}
```
Here is my phpmd.xml:
```xml
<?xml version="1.0"?>
<ruleset name="PHPMD rule set"
         xmlns="http://pmd.sf.net/ruleset/1.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 https://pmd.sf.net/ruleset_xml_schema.xsd"
         xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd"
>
    <description>All rules</description>
    <rule ref="rulesets/cleancode.xml" />
    <rule ref="rulesets/codesize.xml" />
    <rule ref="rulesets/controversial.xml" />
    <rule ref="rulesets/design.xml" />
    <rule ref="rulesets/naming.xml" />
    <rule ref="rulesets/unusedcode.xml" />
</ruleset>
```

## Checks before submitting
* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
* [x] I have added every step to reproduce the bug.
* [x] If possible I added relevant code examples.
* [x] This issue is about 1 bug and nothing more.
* [x] The issue has a descriptive title. For example:  "JSON rendering failed on Windows for filenames with space".

**Comment by tvbeek** [comment]
You get an older version of PHPMD (current is 2.12.0) I have no personal experience with Phive. Is that a problem with Phive or did we miss something?

**Comment by joergmoldenhauer** [comment]
I have installed PHPMD with Phive. I do not get a newer version even if I run `phive update phpmd`.

Maybe the probelm is caused by this older version like describe in #853.

It would be nice if the newest version could be installed with Phive. Is there a Phar with 2.12.0 yet?

**Comment by kylekatarnls** [comment]
Yes: https://github.com/phpmd/phpmd/releases/download/2.12.0/phpmd.phar

You can get them from https://github.com/phpmd/phpmd/releases in the "Assets" section from each release. This is our most up-to-date channel for both composer and phar new releases.

**Comment by jakzal** [comment]
It seems that the signature has not been uploaded to releases since 2.10.0. That's why phive refuses to install versions newer than 2.9.1.

**Comment by joergmoldenhauer** [comment]
I can confirm that. If the `--force-accept-unsigned` option is used the latest version is installed which does not have the described bug. However, this defeats the purpose of using Phive. It would be nice to have a GPG signature for the latest PHPMD version.

**Comment by Xethron** [comment]
@kylekatarnls I believe the release just has to be signed with your GPG key like before. If you look at https://github.com/phpmd/phpmd/releases/tag/2.9.1 you will see it contains the asc file, however, the latest version does not: https://github.com/phpmd/phpmd/releases/tag/2.12.0

This is unfortunately not something we can do, as this requires your GPG key. I did find this HN thread, with a link to the debian wiki on how to do this: https://news.ycombinator.com/item?id=11494997

The top comment suggests a much easier solution of simply signing the release with -s, which in the man pages says:
```
       -s, --sign
           Make a GPG-signed tag, using the default e-mail address’s key. The default behavior of tag GPG-signing is controlled by tag.gpgSign configuration variable if it exists, or disabled otherwise. See git-config(1).
```

I have not tried this myself, but more than happy to look into this further if you need any help.

**Comment by kylekatarnls** [comment]
Thanks.

I'm not sure we'll be able to recover the previous PGP key `pgp@phpmd.org`.

So this will be the new key for signature of the PHAR from 2.10.0:
```
Fingerprint: 9882 DD2B 3881 3C08 EA65 1B69 A4E5 5EA1 2C7C 085C

PHPMD (PHPMD Phar Releases Signer) <phpmd@proton.me>
```

All new releases are now downloadable via Phive.
